scripts/observe_sys_data.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+## Run script command in below format
+## ./scripts/get_sys_observability_data.sh clustername:default containername:wordpress namespace:wordpress-mysql labels:app=mysql
+
+usage()
+{
+    echo "Usage: $0 --containername <name> --clustername <name> --namespace <namespace> --labels <labels> "
+}
+
+OPTS=`getopt -o s: --long containername: --long clustername: --long namespace: --long labels: -n 'parse-options' -- "$@"`
+eval set -- "$OPTS"
+while true; do
+    case "$1" in
+        --containername ) CONTAINER_NAME="$2"; shift 2;;
+        --clustername ) CLUSTER_NAME="$2"; shift 2;;
+        --namespace ) NAMESPACE="$2"; shift 2;;
+        --labels ) LABELS="$2"; shift 2;;
+        -- ) shift; break ;;
+        * ) break ;;
+    esac
+done
+
+[[ "$CLUSTER_NAME" == "" ]] && [[ "$CONTAINER_NAME" == "" ]] && [[ "$NAMESPACE" == "" ]] && [[ "$LABELS" == "" ]] && usage
```

Comment:
why we are printing usage() when no input is provided?

When no input is provided, we should be showing users all the DB entries.